app/_src/gateway/get-started/ai-gateway.md

```diff
@@ -0,0 +1,192 @@
+designed around LLM API use cases. When deployed alongside existing [{{site.base_gateway}} plugins](/hub), 
+{{site.base_gateway}} users can quickly assemble a sophisticated AI management platform 
+without custom code or deploying new and unfamiliar tools.
+
+## Getting Started
+
+The AI Gateway is built using {{site.base_gateway}}'s standard plugin model and the AI Plugins are bundled 
+with {{site.base_gateway}} as of version `3.6`. This means you can deploy the AI Gateway
+capabilities by following the documented configuration instructions for [each plugin](/hub/?category=ai). 
+
+However, to help you get started quickly, we have provided a script that automates the task of 
+deploying a {{site.base_gateway}} configured as an AI Gateway to a Docker container, 
+allowing you to evaluate the core AI provider capabilities before exploring the full suite of AI plugins.
+
+The quickstart script supports deploying {{site.base_gateway}} on {{site.konnect_product_name}} or 
+in traditional mode on Docker only.
+
+{:.note}
+> **Note:**
+> Running this script prompts you for AI Provider API Keys which are used to configure authentication with
+> hosted AI providers. These keys are passed to the {{site.base_gateway}} Docker container and are 
+> not transmitted outside the host machine.
+
+The script runs interactively and prompts you for information about the AI providers you wish to configure.
+
+Download and run script shell in one step:
+
+```sh
+curl -Ls https://get.konghq.com/ai | bash
+```
+
+Once the AI Gateway is deployed, you will see some information displayed to help you evaluate the AI provider 
+proxy behavior. For example, the script will output a sample command helping you route a chat request
+to OpenAI:
+
+```sh
+=======================================================
+ ⚒️                                 Routing LLM Requests
+=======================================================
+
+Your AI Gateway is ready. You can now route AI requests
+to the configured AI providers. For example to route a
+chat request to OpenAI you can use the following
+curl command:
+
+curl -s -X POST localhost:8000/openai/chat \
+ -H "Content-Type: application/json" -d '{
+   "messages": [{
+   "role": "user",
+   "content": "What is Kong Gateway?"
+ }] }'
+```
+
+Additionally, the script will display information about deployment
+files it generates which can be used to for future AI Gateway configurations. 
+
+```sh
+=======================================================
+ ⚒️                What is next with the Kong AI Gateway
+=======================================================
+
+This script demonstrated the installation and usage
+of only one of the many AI plugins that Kong Gateway
+provides (the 'ai-proxy' plugin).
+
+See the output directory to reference the files
+used during the installation process and modify for
+your production deployment.
+ℹ /tmp/kong/ai-gateway
+```
+
+## AI Gateway Capabilities 
```

Comment:
Thanks @cloudjumpercat. I think i'd prefer to keep this information here mostly because I want the document to be about "getting started" with the AI functions.  I don't want the instructions for running the quickstart script to fall 'below the fold'.   I do agree that this could be very valuable information, so maybe it belongs in a different page, or replicated in another place that is more about what AI gateway can do instead of added to the end of this get started page.  Open to ideas.

Reply:
@rspurgeon Hmm, that sounds like a good idea. You could split the "about" info into an Overview page and then have the "how to" info on it's own Get Started page.

Reply:
Do we have a good place for an Overview page ?

Reply:
Maybe in the "Kong in Production" section or the "Kong Plugins" section?